### PR TITLE
[TACHYON-935] Added the usage of links in the documentation for some of the tests in the tachyon-servers module

### PR DESCRIPTION
**Pull request by HorizonNet**
https://tachyon.atlassian.net/browse/TACHYON-935

### CODE REVIEW

File: servers/src/test/java/tachyon/worker/block/evictor/PartialLRUEvictorTestBase.java [hunk 1]
```diff
@@ -23,8 +23,9 @@
 import tachyon.worker.block.TieredBlockStoreTestUtils;
 
 /**
- * Sanity check on specific behavior of PartialLRUEvictor such as evicting/moving least recently
- * used blocks in StorageDir with max free space and cascading ParitialLRU eviction.
+ * Sanity check on specific behavior of {@link PartialLRUEvictor} such as evicting/moving least
+ * recently used blocks in {@link tachyon.worker.block.meta.StorageDir} with max free space and
+ * cascading PartialLRU eviction.
```
Comment (aaudiber) [inline servers/src/test/java/tachyon/worker/block/evictor/PartialLRUEvictorTestBase.java]: Should PartialLRU be `@link`ed?

Comment (HorizonNet) [inline servers/src/test/java/tachyon/worker/block/evictor/PartialLRUEvictorTestBase.java]: @aaudiber Thanks for the comment. I updated the PR accordingly.

**Comment by aaudiber** [comment]
Thanks for the quick iteration!

**Comment by aaudiber** [comment]
+1 LGTM

**Pull request merged**